lib/libfrr.c

```diff
@@ -902,11 +902,14 @@ static int frr_config_read_in(struct thread *t)
 	 * reading the configuration file.
 	 */
 	if (frr_get_cli_mode() == FRR_CLI_TRANSACTIONAL) {
+		struct nb_context context = {};
+		char errmsg[BUFSIZ] = {0};
```

Comment:
What are we doing with return error message we got?

Reply:
yeah we better log what went wrong here too. Fixed.